src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -8,19 +8,23 @@
 using System.Web.Http;
 using Autofac;
 using Autofac.Integration.WebApi;
+using Microsoft.Azure.WebJobs.Script.Settings;
 using Microsoft.Azure.WebJobs.Script.WebHost.Controllers;
 using Microsoft.Azure.WebJobs.Script.WebHost.Handlers;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost
 {
     public static class WebApiConfig
     {
+        private static ISettingsManager _settingsManager;
+
         public static void Register(HttpConfiguration config)
         {
-            Register(config, GetDefaultSettings());
+            _settingsManager = ScriptSettingsManager.Instance;
```

Comment:
this line isn't needed, since the other method sets this. good to keep that centralized